<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectingInfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>connecting...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="mainMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>main menu</string>
    </property>
    <widget class="QLineEdit" name="nickEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>nick...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="joinBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="placeholderText">
      <string>code...</string>
     </property>
    </widget>
    <widget class="QLabel" name="nickLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>holder</string>
     </property>
    </widget>
    <widget class="QLabel" name="codeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>holder</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="creatingQuiz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>creating quiz</string>
    </property>
    <widget class="QPushButton" name="nextBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>quiz name...</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="answerTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="reducedTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="aTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>time to answer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>82</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>reduced time:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="creatingAddQuiz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>creating quiz</string>
    </property>
    <widget class="QPushButton" name="nextAddBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelAddBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="questionEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>70</number>
     </property>
     <property name="placeholderText">
      <string>question...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ansAedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>answer 'A'...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ansBedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>answer 'B'...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ansCedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>answer 'C'...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ansDedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>answer 'D'...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="answerBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="rightAnswerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>192</y>
       <width>53</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>right answer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="ip">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="placeholderText">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="lobby">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="title">
     <string>lobby</string>
    </property>
    <widget class="QPushButton" name="code">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="copyBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameCodeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>game code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="nickBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gameCodeLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>players nicknames</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>322</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
